<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
    <!-- Call agent bringup node -->
    <node name="mtg_agent_bringup_node" pkg="mtg_agent_bringup" type="mtg_agent_bringup_node" output="screen"/>
</launch>
